source/_components/csv.markdown

```diff
@@ -0,0 +1,94 @@
+---
+layout: page
+title: "CSV History"
+description: "Instructions on how to store events in CSV files."
+date: 2018-10-09 21:13
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: History
+ha_release: 0.80
+---
+
+The `csv` component lets you store all state changes inside CSV files. 
+
+To use this component in your installation, add the following to your `configuration.yaml` file:
+
+## {% linkable_title Configuration %}
+
+```yaml
+# Example configuration.yaml entry
+csv:
+  data_dir: '/home/homeassistant/data_dir'
+  separator: ','
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.